substrate/coins/primitives/src/lib.rs

```diff
@@ -12,7 +12,7 @@ use scale_info::TypeInfo;
 
 use serai_primitives::{Balance, SeraiAddress, ExternalAddress, Data, pallet_address};
 
-pub const ADDRESS: SeraiAddress = pallet_address(b"Tokens");
+pub const FEE_ACCOUNT: SeraiAddress = pallet_address(b"FeeAccount");
```

Comment:
Should we rename pallet_address to system_address?

Reply:
makes sense